### DISCUSSION TITLE
11ty new logo

### DISCUSSION THREAD

**Discussion by lovedder1995** [11ty/eleventy#1476] [Show and tell]
Is a bolt of lightning using two numbers one

![11ty](https://user-images.githubusercontent.com/29613190/97215584-41b15b00-178a-11eb-9b55-2f3a7474ec5b.png)

What do you think about?

**Comment by Ryuno-Ki** [comment]
Don't take it personal, but since you've asked for feedback, let me share some thoughts.

Without you mentioning it, I wouldn't have seen the two 1s.
Perhaps it would be more visible if the vertical lines would go a bit more towards the vertex?

How does it look like against a dark/light background? Is there enough contrast?
How would it look like resized to 16x16 px (favicon)?
How would it look like on Social Media (Twitter, Discord etc.)
How would it look like on a t-shirt? On a mug? (Merchandising / spreading the word)

I don't ask you to provide an image for all those questions. But perhaps it helps thinking a bit more about what goes into a logo.

**Comment by lovedder1995** [comment]
I won't take it personally, after all it's about getting things done right.

Also no one is requesting a logo, so it's not about impose it.

![11ty](https://user-images.githubusercontent.com/29613190/97219267-5b08d600-178f-11eb-9870-137c62c819af.png)
![11ty](https://user-images.githubusercontent.com/29613190/97219476-b0dd7e00-178f-11eb-9d66-b53d3820cab2.png)
![11ty](https://user-images.githubusercontent.com/29613190/97219369-87bced80-178f-11eb-9d52-4b96c111d735.png)

The two 1s are now visible?

I work with vinyl every day, that's what I do. On a t-shirts will look well, and can be printed using any printing technique.

The yellow is a neutral tone, is visible against dark and light background, but I prefer the dark background.

**Reply by Ryuno-Ki** [reply]
![logo](https://user-images.githubusercontent.com/3097194/97233062-9979b180-17de-11eb-99b0-f01174ce55ce.png)

I quickly hacked the PNG. So I'm aware, that it looks imbalanced. Something like this makes the 1s stand out more, I think.
Since I already have a dark theme (via Stylus web extension) I couldn't see the image against a white background.
But I trust you that it will look okay.

**Reply by lovedder1995** [reply]
![11ty](https://user-images.githubusercontent.com/29613190/97250991-b0170d00-17cc-11eb-96a5-3ab8d173c5ed.png)

**Comment by lovedder1995** [comment]
I think is better be a little more inclined.

![11ty](https://user-images.githubusercontent.com/29613190/97221168-2c402f00-1792-11eb-8341-4bc6fa2608fc.png)

**Reply by Ryuno-Ki** [reply]
Reminds a bit on a highway now :-D
But well, Eleventy is fast, so that association fits.

Or a car:
![Porsche 911 with two stripes as foil](http://www.car-revs-daily.com/wp-content/uploads/FOSTLA.de-Shows-Artful-Porsche-911-GT3-Foil-Wrap-Design-and-Execution-10.jpg)

**Reply by lovedder1995** [reply]
I tried, but it's very difficult to illustrate a car with the number 11.

The logo would become more complex and therefore could not be printed on any printing technique.

**Comment by lovedder1995** [comment]
If you want to edit the design here is the SVG file.

[11ty.zip](https://github.com/11ty/eleventy/files/5441155/11ty.zip)

**Reply by Ryuno-Ki** [reply]
Perhaps worth to run it through https://jakearchibald.github.io/svgomg/ before zipping (since the visual tools you use add a lot of unnecessary bloat to it).

**Comment by zachleat** [comment]
I do like this but it reminds me too much of AMP 😭
![image](https://user-images.githubusercontent.com/39355/97749055-670cd480-1abc-11eb-8df4-bbb4920e7ee7.png)

**Reply by lovedder1995** [reply]
I found another way to illustrate the 11.

![11ty](https://user-images.githubusercontent.com/29613190/97751503-0e3b3d00-1ab8-11eb-9165-07b84443bb24.jpg)

I think we can more clearly see the 11 and the bolt of lightning.

**Reply by lovedder1995** [reply]
![11ty](https://user-images.githubusercontent.com/29613190/97765711-c2e55680-1ad8-11eb-988a-ae6f11d72639.png)

**Reply by zachleat** [reply]
Honestly I do really appreciate the effort here but I don’t think we are in the market for a new logo at this point.

Just for educational purposes because I assume this was accidental I do feel compelled to let you know that this latest iteration bears a very strong resemblance to the Nazi SS logo. I only say this so you’re aware of the symbol and don’t repeat this mistake in future design endeavors ❤️

All the best, Zach
